routing/localchans/manager.go

```diff
@@ -155,6 +167,30 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 					"not yet confirmed",
 				))
 
+		case createMissingEdge:
+			// If the edge was not found, but the channel is found,
+			// that means the edge is missing in the graph database
+			// and should be recreated. The edge and policy are
+			// created in-memory and will be added to the graph in
+			// the PropagateChanPolicyUpdate call below.
```

Comment:
So my analysis showed we cannot do it this way because `PropagateChanPolicyUpdate ` will not persist the edge info and therefore the UpdateEdge call will fail, either we maybe just create a new ChanAnnoucement to our self here and make sure we don't broadcast it to the network again lets see, I will come back to you with my evaluation.